### PR TITLE
Create Codacy configuration file

### PR DESCRIPTION
**Pull request by abelsiqueira**
**Description**

Create Codacy configuration file

**Related issues**:

- #81

**Instructions to review the pull request**

- Check that CHANGELOG.md has been updated if necessary

Review online.

### CODE REVIEW

**Comment by abelsiqueira** [comment]
I don't see Codacy running, but the configuration was considered valid by the CLI (https://github.com/codacy/codacy-analysis-cli/)

File: .codacy.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+engines:
+  cppcheck:
+    language: c++
+  flawfinder:
+    exclude_paths:
+languages:
+  cpp:
+    extensions:
+      - '*.cpp'
+      - '*.hpp'
```
Comment (fdiblen) [inline .codacy.yml:11]:
```suggestion
      - '*.hpp'
      
```

**Review by fdiblen — APPROVED** [review]
Thanks @abelsiqueira :lab_coat:

**Comment by abelsiqueira** [comment]
Thanks for the review :1234:

**Pull request merged**